src/wp-admin/admin-header.php

```diff
@@ -190,8 +190,7 @@
 $admin_body_class .= ' no-customize-support no-svg';
 
 if ( $current_screen->is_block_editor() ) {
-	// Default to is-fullscreen-mode to avoid jumps in the UI.
-	$admin_body_class .= ' block-editor-page is-fullscreen-mode wp-embed-responsive';
+	$admin_body_class .= ' block-editor-page wp-embed-responsive';
```

Comment:
Widgets wasn't, no. This change (combined with the change in `edit-form-blocks.php`) makes it so that we go to full screen when loading the post and page editor, not when `$current_screen->is_block_editor` is true. This is because the widgets editor is a block editor but not full screen.